### PR TITLE
Labs

### PR DESCRIPTION
**Pull request by hallentilford**
update orb namespace to atlassian-labs

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+[![CircleCI Build Status](https://circleci.com/gh/atlassian-labs/Compass-Orb.svg?style=shield 'CircleCI Build Status')](https://circleci.com/gh/atlassian-labs/Compass-Orb) [![CircleCI Orb Version](https://badges.circleci.com/orbs/atlassian-labs/compass.svg)](https://circleci.com/orbs/registry/orb/atlassian/compass) [![Atlassian license](https://img.shields.io/badge/license-Apache%202.0-blue.svg?style=flat-square)](LICENSE) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)](CONTRIBUTING.md)
 
 An orb for reporting the status of CircleCI builds and deployments to Atlassian Compass
 
 ## Usage
 
-For full usage guidelines, see the [orb registry listing](http://circleci.com/orbs/registry/orb/atlassian/compass).
+For full usage guidelines, see the [Intregrate Compass with CircleCI guide](https://developer.atlassian.com/cloud/compass/integrations/integrate-Compass-with-CircleCI/).
```
Comment (mdellandrea) [inline Readme.md:9]: Should we use the go link instead here? https://go.atlassian.com/compass-circleci-integration

File: src/examples/basic_deployment.yml [hunk 2]
```diff
@@ -4,17 +4,16 @@ description: |
 usage:
```
Comment (kbielaski) [inline src/examples/basic_deployment.yml:4]: We'll probably also want to update our instructions in the app config page

**Review by kbielaski — APPROVED** [review]

**Comment by hallentilford** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/atlassian-labs/compass?version=dev:751d405ddbf229596d0008f8c95dda9e39264b15

**Review by mdellandrea — APPROVED** [review]

**Comment by hallentilford** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/atlassian-labs/compass?version=dev:56423ed06ed9e1db9bd04d187ff4816474814577

**Comment by hallentilford** [comment]
Your development orb has been published. It will expire in 30 days.
You can preview what this will look like on the CircleCI Orb Registry at the following link: 
https://circleci.com/developer/orbs/orb/atlassian-labs/compass?version=dev:56423ed06ed9e1db9bd04d187ff4816474814577

**Pull request merged**